source/_components/plant.markdown

```diff
@@ -11,9 +11,14 @@ ha_category: Other
 ha_release: 0.44
 ---
 
+This`plant`component lets you merge moisture, conductivity, light intensity, temperature and battery level for a plant into a single UI element. It also supports setting minimum and maximum values for each measurement and will change its state to "problem" if it is not within those limits.
+
+To use your `plant` sensor in your installation, add the following to your `configuration.yaml` file:
+
 ```yaml
+# Example configuration.yaml entry
 plant:
-  simulated_plant:
+  entity_id:
```

Comment:
Since this is an example we should use a real plant name instead. This might be confusing.